### ISSUE TITLE
Bug Report

### ISSUE THREAD

**Issue by Jesse-XIE** [rinuboney/ladder#17]
In ladder.py, line 179, you normalized z_est by:
 ` z_est_bn = (z_est[l] - m) / v`
m, v are calculated in line 92, 
 `m, v = tf.nn.moments(z_pre_u, axes=[0])`
which means that v is variance, **not** the standard deviation. So line 179 should be modified to:  `z_est_bn = (z_est[l] - m) / tf.sqrt(v + epsilon)`   
where epsilon is a small number.
Hope that helps.
